packages/suite/src/middlewares/wallet/walletMiddleware.ts

```diff
@@ -96,6 +101,33 @@ const walletMiddleware =
             api.dispatch(coinmarketBuyActions.dispose());
         }
 
+        if (
+            prevState.router.pathname !== '/accounts/send' &&
+            nextRouter.pathname !== '/accounts/send' &&
+            action.type === WALLET_SETTINGS.SET_BITCOIN_AMOUNT_UNITS
+        ) {
+            const { selectedAccount, settings } = api.getState().wallet;
+            const formattedDraft: FormState = { ...api.dispatch(sendFormActions.getDraft()) };
+
+            if (selectedAccount.account && formattedDraft && formattedDraft.outputs) {
```

Comment:
No account is selected while user is in settings. Also it makes no sense to me apply it on selected account only - you could have drafts on multiple accounts at the same time.

Reply:
Right, then then there is no selected acc on the dashboard either.

Reply:
fa39d3319c7953ea6e6d39c30bc33142d7481bb8